fdbserver/VersionedBTree.actor.cpp

```diff
@@ -7326,8 +7326,11 @@ class KeyValueStoreRedwood : public IKeyValueStore {
 		                   : FLOW_KNOBS->SIM_PAGE_CACHE_4K)
 		        : FLOW_KNOBS->PAGE_CACHE_4K;
 		// Rough size of pages to keep in remap cleanup queue before being cleanup.
-		int64_t remapCleanupWindowBytes = (BUGGIFY ? deterministicRandom()->randomInt64(0, 100) * 1024 * 1024
-		                                           : SERVER_KNOBS->REDWOOD_REMAP_CLEANUP_WINDOW_BYTES);
+		int64_t remapCleanupWindowBytes = SERVER_KNOBS->REDWOOD_REMAP_CLEANUP_WINDOW_BYTES;
+		if (g_network->isSimulated() && BUGGIFY)
```

Comment:
This logic won't change the value from the knob default for non-buggified simulation runs.  Also, if you multiple the max passed to the random int generator the resulting parameters will be less "round" which could be useful.
For example,
```
int64_t remapCleanupWindowBytes = g_network->isSimulated() ? 100 * 1024 * 1024 : SERVER_KNOBS->REDWOOD_REMAP_CLEANUP_WINDOW_BYTES;
if (BUGGIFY) {
  remapCleanupWindowBytes = deterministicRandom()->coinflip() ? deterministicRandom()->randomInt64(0, 100 * 1024) : deterministicRandom()->randomInt64(0, 100 * 1024 * 1024);
}
```